packages/components/src/unit-control/stories/index.tsx

```diff
@@ -54,16 +51,14 @@ const DefaultTemplate: ComponentStory< typeof UnitControl > = ( {
 	const [ value, setValue ] = useState< string | undefined >( '10px' );
 
 	return (
-		<div style={ { maxWidth: '100px' } }>
```

Comment:
Removed these max-width containers in the stories because we need to know whether the width is unconstrained by default, and we need to be able to test `__unstableInputWidth`.